docs/data/toolpad/core/components/sign-in-page/BrandingSignInPage.js

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { AppProvider, SignInPage } from '@toolpad/core';
+import { createTheme } from '@mui/material/styles';
+import { useColorSchemeShim } from 'docs/src/modules/components/ThemeContext';
+import { getDesignTokens } from './brandingTheme';
+
+const providers = [
+  { id: 'github', name: 'GitHub' },
+  { id: 'google', name: 'Google' },
+  { id: 'credentials', name: 'Email and Password' },
+];
+
+const BRANDING = {
```

Comment:
Maybe these values being passed should be shown in the preview if that doesn't make the preview too complex or take up too much space.